### ISSUE TITLE
U+8AF2 諲 in Korean subset OTFs

### ISSUE THREAD

**Issue by dine2014** [adobe-fonts/source-han-serif#99]
### Prerequisites

* [x] If you are reporting an issue that affects glyphs for characters for a particular region or regions, did you verify that the characters are within the supported scope of the region or regions? This generally means GB 18030 or Tōngyòng Guīfàn Hànzìbiǎo (通用规范汉字表) for China, Big Five or CNS 11643 Planes 1 & 2 for Taiwan, HKSCS-2016 for Hong Kong (though support has been deferred until Version 2.000), the JIS standards for Japan, and KS X 1001 and KS X 1002 for Korea.
* [x] Did you thoroughly search the [open](https://github.com/adobe-fonts/source-han-serif/issues) and [closed](https://github.com/adobe-fonts/source-han-serif/issues?q=is%3Aissue+is%3Aclosed) issues to avoid reporting a duplicate issue?
* [x] Did you go through the [official font readme file](https://github.com/adobe-fonts/source-han-serif/raw/release/SourceHanSerifReadMe.pdf) to better understand the scope of the project, to include the *Known Issues* section at the very end?

### Description

The Korean subset OTFs have U+8AF2 諲 in Chinese style (the first stroke being a dot).

**Comment by punchcutter** [comment]
This isn't going to change right now because the only non-Chinese style glyph has been removed to make room for Hong Kong support.

**Comment by dine2014** [comment]
So in the next release U+8AF2 諲 will be removed from the Korean subset OTFs, rather than using the Japanese glyph?

**Comment by punchcutter** [comment]
No, the JP glyph has been removed so the Korean glyph will still map to the CN glyph, the same as in Source Han Sans.

**Comment by dine2014** [comment]
I see.

**Comment by punchcutter** [comment]
I should have said TW glyph. The CN glyph is different so KR will map to the TW glyph as the most similar.

**Comment by punchcutter** [comment]
@LsnmxNB There haven't been any updates since then, but when we have a new release ready it will be updated.

**Comment by LsnmxNB** [comment]
> @LsnmxNB There haven't been any updates since then, but when we have a new release ready it will be updated.

OK. Thank you
It’s been too long and I’m looking forward to your release
